sdk/keyvault/keyvault-secrets/samples-dev/helloWorld.ts

```diff
@@ -24,6 +24,9 @@ export async function main(): Promise<void> {
   const client = new SecretClient(url, credential);
 
   // Create a secret
+  // The secret can be a string, a multiline text block such as
```

Comment:
```suggestion
  // The secret can be a string of any kind. For example, a multiline text block such as
```
What do you think of this suggestion?